lib/import-notebook.js

```diff
@@ -77,18 +113,34 @@ export async function _loadNotebook(filename: string) {
   const editor = await atom.workspace.open();
   const grammar = getGrammarForNotebook(nb);
   if (!grammar) return;
-  editor.setGrammar(grammar);
+  atom.grammars.assignLanguageMode(editor, grammar.scopeName);
```

Comment:
What's the reason for this change ?
It seems to fail to recognise Julia notebook as julia file in my local test.

Reply:
Hmm, but I found the issue only happens on this PR, and not on master thus I suspect the change is causing something bad.

Reply:
Maybe for `getGrammarForNotebook` work, we still need the `setGrammar`.
You didn't face the issue maybe because Python grammar is handled with hard code.

***

Rather off topic, but I would really really
appreciate if you also test with various kernels other than IPython in general.

Reply:
You are probably right and I believe that getGrammarForNotebook may need to be updated too.

Reply:
Fixed